### PR TITLE
fix

### PR DESCRIPTION
**Pull request by Tinchuu**

### CODE REVIEW

File: project/src/utils/test_train_split.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import os
+from glob import glob
+from pathlib import Path
+
+from sklearn.model_selection import train_test_split
+
+from utils.utils import create_dir, open_csv, to_csv
+
+
+def split_data(path_data, path_output, test_size):
```
Comment (luorixo) [inline project/src/utils/test_train_split.py]: Can you rename this to `test_train_split` or something more descriptive 🙏

File: project/.gitignore [hunk 2]
```diff
@@ -10,6 +10,9 @@ __pycache__/
 .Python
 data
 build/
+split
+results
+result
```
Comment (luorixo) [inline project/.gitignore:15]: Can remove if you can be bothered to

**Review by luorixo — APPROVED** [review]
Looks good, If you can be bothered, you can make the requested changes though it's not blocking.

File: project/.gitignore (hunk 2 above)
Comment (Tinchuu) [reply project/.gitignore:15]: remove what

**Pull request merged**